src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -373,7 +376,7 @@ def init_sync_d_queue_tables(db_conn):
 
     # Parse the queue_name_map and create the following maps:
     # port_queues_map -> {"if_index : queue_index" : sai_oid}
-    # queue_stat_map -> {queue stat table name : {counter name : value}}
+    # queue_stat_map -> {"if_index : queue stat table name" : {counter name : value}} 
```

Comment:
Please add some comment as why these changes are needed (OID overlap) in Multi-Asic platforms

Reply:
Updated as per comment.